oanda_v20_platform_public.egg-info/SOURCES.txt

```diff
@@ -0,0 +1,23 @@
+setup.py
```

Comment:
Its generated by running setup.py. And more information can be added to that file. However, I was reading the docs for setuptools and say that setup.py is being depreciated in favour of setup.cfg   
https://setuptools.readthedocs.io/en/latest/userguide/quickstart.html
So I guess it would be best to go down that route. Or maybe Github actions? I haven't got much experience with creating deployable packages, so this is a bit trial and error for me! I envisaged making an easy package to install on a VM, but then being able to update it with the latest version, say at the weekend - then carry on trading when the markets reopen. Is that what your thinking too?

Reply:
And pip install -e .

Reply:
nice, i'll be reading up on those.  I don't have much experience with deployable packages either. so TBH I'm not sure what direction to go with that. right now I just have some bash scripts that I use to create a new venv on my server, pull the code from a repo, and start it up using nohup. Not very 'production grade' but I'm still practicing and I've deployed hundreds of times and had some of them running for 6 months without failures.  I'd love to have versioned packages that I can deploy through a script though so its more robust though.